src/Middleware/BasicAuthMiddleware.php

```diff
@@ -1,11 +1,11 @@
 <?php
 
-namespace Dumbo\Helpers;
+namespace Dumbo\Middleware;
 
 use Dumbo\Context;
 use Psr\Http\Message\ResponseInterface;
 
-class BasicAuth
+class BasicAuthMiddleware
```

Comment:
@lucasacoutinho should we omit `Middleware` from the names here? It might look cleaner when using them? Wdyt